.circleci/config.yml

```diff
@@ -142,4 +141,5 @@ jobs:
           cluster-name: 'Book-A-Desk-Cluster'
           container-image-name-updates: 'container=Book-A-Desk-Container,tag=<< pipeline.parameters.docker-img-version >>'
           service-name: 'Book-A-Desk'
-          family: 'Book-A-Desk-Backend'
\ No newline at end of file
+          family: 'Book-A-Desk-Backend'
+          container-env-var-updates: container=Book-A-Desk-Container,name=AWS_KEYID,value=$AWSKEYID,container=Book-A-Desk-Container,name=AWS_SECRETKEY,value=$AWSSECRETKEY,container=Book-A-Desk-Container,name=ENVIRONMENT,value=$ENVIRONMENT
```

Comment:
environment variables sent to ECS because the image will not be rebuilt, different environment variables with be sent depending on stages